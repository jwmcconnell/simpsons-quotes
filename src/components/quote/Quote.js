import React from 'react';
import PropTypes from 'prop-types';

const Quote = ({ quote, characterName, characterImage }) => {
  return (
    <section>
      <h2>{characterName}</h2>
      <p>{quote}</p>
      <img src={characterImage}></img>
    </section>
  );
};

Quote.propTypes = {
  quote: PropTypes.string.isRequired,
  characterImage: PropTypes.string.isRequired,
  characterName: PropTypes.string.isRequired
};

export default Quote;
