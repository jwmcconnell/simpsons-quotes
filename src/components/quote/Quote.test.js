import React from 'react';
import { shallow } from 'enzyme';
import Quote from './Quote';

describe('Quote component', () => {
  it('renders Quote', () => {
    const wrapper = shallow(<Quote 
      quote=""
      characterName=""
      characterImage=""
    />);
    expect(wrapper).toMatchSnapshot();
  });
});
  
