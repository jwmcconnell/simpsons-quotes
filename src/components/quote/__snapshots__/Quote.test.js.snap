// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote component renders Quote 1`] = `
<section>
  <h2 />
  <p />
  <img
    src=""
  />
</section>
`;
